modules/@apostrophecms/page-type/index.js

```diff
@@ -497,6 +545,14 @@ module.exports = {
           }
           return label;
         }
+      },
+      getBrowserData(_super, req) {
+        const browserOptions = _super(req);
+
+        browserOptions.filters = self.filters;
+        browserOptions.columns = self.columns;
+
```

Comment:
Move more code from `piece-type` to set up `browserOptions.components` here and then you won't have to hardcode it on the front end

Reply:
Done!